### PR TITLE
Added Spring Security

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

File: src/main/java/ua/antibyte/cinema/model/dto/UserRegistrationDto.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import lombok.Data;
+import ua.antibyte.cinema.annotation.EmailValidation;
+import ua.antibyte.cinema.annotation.FieldsValueMatch;
+
+@Data
+
+
```
Comment (AnaLanda) [inline src/main/java/ua/antibyte/cinema/model/dto/UserRegistrationDto.java]: Remove the empty lines here)

**Review by AnaLanda — COMMENTED** [review]
Looks great!

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job!

File: src/main/java/ua/antibyte/cinema/annotation/FieldsValueMatchValidator.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext constraintValidatorContext) {
+        Object fieldValue = new BeanWrapperImpl(value).getPropertyValue(filed);
+        Object fieldMatchValue = new BeanWrapperImpl(value).getPropertyValue(fieldMatch);
+
+        if (fieldValue != null) {
+            return fieldValue.equals(fieldMatchValue);
+        } else {
+            return fieldMatchValue == null;
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/ua/antibyte/cinema/annotation/FieldsValueMatchValidator.java:25]: we can replace it with ternary operator

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**